plone/app/querystring/profiles/upgrades/to_9/registry.xml

```diff
@@ -0,0 +1,44 @@
+<registry xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+          i18n:domain="plone">
+
+    <records interface="plone.app.querystring.interfaces.IQueryField"
+             prefix="plone.app.querystring.field.portal_type">
+        <value key="title" i18n:translate="">Type</value>
+        <value key="description" i18n:translate="">An item's type (e.g. Event)</value>
+        <value key="enabled">True</value>
+        <value key="sortable">False</value>
+        <value key="operations">
+            <element>plone.app.querystring.operation.selection.any</element>
+        </value>
```

Comment:
Would it not be enough to set the new `<value key="operations">` and leave the rest out or is that then purged?